Example/android/app/build.gradle

```diff
@@ -132,6 +132,8 @@ dependencies {
     } else {
         implementation jscFlavor
     }
+
+    implementation("com.google.android.material:material:1.9.0")
```

Comment:
I'm worried if anyone that will try to install the Example app on Android will stumble upon the issue with too low version of Kotlin. Should we maybe add some comment here or create a README with prerequisites to install the app?